src/ci/docker/host-x86_64/x86_64-gnu-tools/checktools.sh

```diff
@@ -2,6 +2,7 @@
 # ignore-tidy-linelength
 
 set -eu
+set -x # so one can see where we are in the script
```

Comment:
I think this is something we may want to keep, just to have a bit more info in the logs.